deploy/vagrant/conf/gce.yml.template

```diff
@@ -0,0 +1,69 @@
+# Setting up your Google Cloud Project
+#
+# The Project ID for your Google Cloud Platform account.
+Google_Cloud_Project_ID:
+
+# The Client Email address for your Service Account.
+Service_Acount_Email_Address:
+
+# The location of the JSON private key file matching your Service Account.
+json_Key_Location: 
+
+
+#SSH - Vagrant/Ansible will SSH into VMs for environment setup.
+#
+# This is the path to SSH keys. The easiest way to setup GCE keys is using the 
+# CLI (gcloud compute config-ssh) Which creates metatdata that is provided to 
+# every VM launched. Alternatively, it is possible to add to the GCE metatdata service
+# directly or manually manage keys.
+Key_Path: "~/.ssh/google_compute_engine"
```

Comment:
I ran into a issue using "~/.ssh/google_compute_engine". But switching to absolute path, "/Users/chaomin/.ssh/google_compute_engine" would work. Is this related with vagrant parsing path? Would you suggest using absolute path here?

=== Logs====
{"Google_Cloud_Project_ID"=>"corded-guild-125817", "Service_Acount_Email_Address"=>"chaomin@alluxio.com", "json_Key_Location"=>"~/.ssh/Project-12e448f09873.json", "Key_Path"=>"~/.ssh/google_compute_engine", "SSH_Username"=>"chaomin", "Preemptible"=>false, "Machine_Type"=>"n1-standard-1", "Scopes"=>["https://www.googleapis.com/auth/cloud-platform"], "Disk_Size"=>10, "Tags"=>"Alluxio", "Name"=>nil, "Network"=>"default", "Image"=>"centos-7-v20160301"}
There are errors in the configuration of this machine. Please fix
the following errors and try again:

Google Provider:
* Private key for Google could not be found in the specified location.
======